buildSrc/src/main/kotlin/dependencies/Dependencies.kt

```diff
@@ -21,6 +21,7 @@ object Dependencies {
         const val daggerAssistedInject = "0.3.2"
         const val expandableRecyclerView = "3.0.0-RC1"
         const val billing = "1.2"
+        const val apache = "lang3:3.0"
```

Comment:
I think `lang3:` should really be part of the below library name, and this variable should just be `3.0`, if `3.0` is the version number.